tuist/tuist
# Add ability to pass specified environment keys when calling Tuist.graph(at:)

Resolves https://github.com/tuist/tuist/issues/5125

### Short description 📝

This PR adds ability to pass environment keys when we call `Tuist.graph(at:)` method.

### How to test the changes locally 🧐

Create a `Tuist` plugin that uses `Tuist.graph(at:)` internally, and call this plugin through the `tuist` scheme and pass some environment variables, then print out the `ProcessInfo.processInfo.environment` to see whether these environment keys passed to the relevant process.

### Contributor checklist ✅


### Reviewer checklist ✅
